test/geometry/transform/test_thin_plate_spline.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import kornia as kornia
+import kornia.testing as utils
+
+import torch
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+
+
+class TestTpsParameters:
+    @pytest.mark.parametrize("batch_size", [1, 2, 3])
+    @pytest.mark.parametrize("device", ('cpu', 'cuda'))
+    def test_smoke(self, batch_size, device):
```

Comment:
you need to add `dtype` to the signature so that can be propagated throw the functional